edgelet/edgelet-docker/src/settings.rs

```diff
@@ -308,7 +308,7 @@ impl From<ErrorKind> for LoadSettingsError {
 
 #[cfg(test)]
 mod tests {
-    #[cfg(target_os = "linux")]
+    #[cfg(unix)]
```

Comment:
If you're changing it you can just remove it entirely. All the code in this branch is Linux-only now.

Reply:
Sweet, all in favour of removing unnecessary conditional compilation :)